public/js/index.js

```diff
@@ -1786,6 +1787,14 @@ function initVueComponents(){
     })
 }
 
+function initCtrlEnterSubmit() {
+    $("textarea").keydown(function(e) {
+        if ((e.ctrlKey || e.metaKey) && (e.keyCode == 13 || e.keyCode == 10)) {
```

Comment:
This will not work on windows correctly as e.ctrlKey will fire also on AltGr+Enter. It should be something like:
`(e.ctrlKey && !e.altKey) || e.metaKey` to check if ctrl/meta key is pressed